dashel/dashel-win32.cpp

```diff
@@ -311,7 +311,7 @@ namespace Dashel
 			if (bindAddress.port == 0)
 			{
 				int addrSize(sizeof(addr));
-				if (getsockname(fd, (struct sockaddr *)&addr, &addrSize) != 0)
+				if (getsockname(sock, (struct sockaddr *)&addr, &addrSize) != 0)
```

Comment:
In 0686c134a48c858cc4aff86bb7c9c4827f0c9b75 the variable name from dashel-posix.cpp was used here, breaking compilation